apps/book/models.py

```diff
@@ -62,6 +62,24 @@ class LanguageType(models.TextChoices):
         NEPALI = 'nepali', 'Nepali'
         ENGLISH = 'english', 'English'
 
+    class Grade(models.TextChoices):
+        GRADE_ONE = 'grade_one', 'Grade one'
+        GRADE_TWO = 'grade_two', 'Grade two'
+        GRADE_THREE = 'grade_three', 'Grade three'
+        GRADE_FOUR = 'grade_four', 'Grade four'
+        GRADE_FIVE = 'grade_five', 'Grade five'
+        GRADE_SIX = 'grade_six', 'Grade six'
+        GRADE_SEVEN = 'grade_seven', 'Grade seven'
+        GRADE_EIGHT = 'grade_eight', 'Grade eight'
+        GRADE_NINE = 'grade_nine', 'Grade nine'
+        GRADE_TEN = 'grade_ten', 'Grade ten'
+        GRADE_ELEVEN = 'grade_eleven', 'Grade eleven'
+        GRADE_TWELVE = 'grade_twelve', 'Grade twelve'
```

Comment:
```suggestion
        GRADE_ONE = 'grade_one', _('Grade one')
        GRADE_TWO = 'grade_two', _('Grade two')
        GRADE_THREE = 'grade_three', _('Grade three')
        GRADE_FOUR = 'grade_four', _('Grade four')
        GRADE_FIVE = 'grade_five', _('Grade five')
        GRADE_SIX = 'grade_six', _('Grade six')
        GRADE_SEVEN = 'grade_seven', _('Grade seven')
        GRADE_EIGHT = 'grade_eight', _('Grade eight')
        GRADE_NINE = 'grade_nine', _('Grade nine')
        GRADE_TEN = 'grade_ten', _('Grade ten')
        GRADE_ELEVEN = 'grade_eleven', _('Grade eleven')
        GRADE_TWELVE = 'grade_twelve', _('Grade twelve')
```

Reply:
@thenav56  Verbose name changed

Reply:
@bimal125 Please re-read the message above.